builders/build-python.ps1

```diff
@@ -49,13 +50,13 @@ function Get-PythonBuilder {
 
     #>
 
-    param (
-        [version] $Version,
+    param(
+        [semver] $Version,
         [string] $Architecture,
         [string] $Platform
     )
 
-    $Platform = $Platform.ToLower()  
+    $Platform = $Platform.ToLower()
```

Comment:
By default, all comparison operators are case-insensitive. You don't need this statement.

Reply:
Thanks!